qcodes/dataset/measurements.py

```diff
@@ -412,6 +413,17 @@ def flush_data_to_database(self, block: bool = False) -> None:
         """
         self.dataset._flush_data_to_database(block=block)
 
+    def export_data(self) -> bool:
+        """Export data at end of measurement if export_type
+        is specified in "dataset" config
+
+        Returns:
+            bool: Flag to export data
+        """
+        export_type = get_data_export_type()
+        if export_type is not None:
+            self.dataset.export(export_type=export_type)
```

Comment:
This does not return any thing so the signature does not match

Reply:
Thanks for pointing that out, I've removed the `-> bool` type hint.